Source/FullScreenPokemon.ts

```diff
@@ -8607,7 +8615,12 @@ module FullScreenPokemon {
         }
 
         /**
+         * Adds an in-game item to the character's bag
+         * using localStorage.
```

Comment:
You can remove "using localStorage". In general, localStorage shouldn't be mentioned because ItemsHoldr and StateHoldr are abstractions on top of it. 

In the future, it's possible something else like indexedDB would be used instead.